app/helpers/campaigns_helper.rb

```diff
@@ -0,0 +1,22 @@
+module CampaignsHelper
+  def substituted_content(campaign, rsvp)
+    content = substitute_content campaign.content, rsvp
+
+    markdown_to_html content
+  end
+
+  # rubocop:disable Rails/OutputSafety
+  def substitute_content(content, rsvp)
+    return content if rsvp.nil?
+
+    content.gsub(
+      '{{ rsvp_links }}',
+      tag.p(
```

Comment:
does this render as-expected? reading this I'd guess it'd render the `button` class on the paragraph such that:

`[Yes No]` instead of `[Yes] [No]`

sidenote: I didn't know about the new syntax content_tag helper! nice :) (apparently a 5.1+ feature)

Reply:
I didn't know `content_tag` was deprecated either until I was reading the docs last night :)

And the buttons do appear separately: 
![Screen Shot 2019-10-08 at 9 50 15 am](https://user-images.githubusercontent.com/4183/66354611-1a39f580-e9b1-11e9-80a5-33d1d7e3b58f.png)